app/views/shared/_paginator.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="paginator">
+  <ul class="pagination">
```

Comment:
Nice! I like the idea of using a list here. But I think if we are going to do so we might want to use an `ol`.
This is a nitpicky comment... but a list of ordered numbers strikes me as list where order matters.